### DISCUSSION TITLE
Deployed to Azure, something changed

### DISCUSSION THREAD

**Discussion by mjohnson-xbridge-io** [microsoft/FHIR-Converter#137] [Q&A: Handlebars Engine] [answered]
A few months ago, I deployed the fhir converter service to Azure, started using the service, including some new templates we created. A few days ago, something happened with our service and our custom templates were no longer available in the service.

Previous to that time, we could find the templates, using ftp to connect to the Azure service, in a folder structure starting with `fhirconvertertemplates/hl7v2/`. Adding templates either through ftp or through the service's UI accomplished the same thing. Now, we can still see those files through ftp, but they're not available in the service UI or when we call the service api to convert documents.

I have 2 questions:
1. is there a change that happened in this github repo that could have affected our Azure service?
2. has the storage structure of the service changed so that the templates are no longer stored in the file structure we were using

Thanks for any help or guidance!

--Mark

**Comment by ranvijaykumar** [comment, marked as answer]
This issue got introduced in one of our updates. This link should help you:
https://github.com/microsoft/FHIR-Converter/blob/handlebars/docs/web-ui-auto-update-issue.md

**Reply by mjohnson-xbridge-io** [reply]
Should the advice in that article still work? I'm attempting to use healthplatformregistry.azurecr.io/fhirconverter:v1.0.0 and am getting the following error message in my logs:

{"message":"manifest for healthplatformregistry.azurecr.io/fhirconverter:v1.0.0 not found: manifest unknown: manifest tagged by \"v1.0.0\" is not found"}

**Reply by ranvijaykumar** [reply]
Thanks for reporting this issue. We are investigating this.

**Reply by ranvijaykumar** [reply]
Can you please verify the behavior? We have applied the fix for it.
